std/js/_std/haxe/iterators/StringIteratorUnicode.hx

```diff
@@ -45,9 +45,8 @@ class StringIteratorUnicode {
 	/**
 		See `Iterator.next`
 	**/
-	@:access(StringTools)
 	public inline function next() {
-		var c = StringTools.utf16CodePointAt(this.current.value, 0);
+		var c = (cast this.current.value).codePointAt(0);
```

Comment:
This is not equivalent. Have a look at the `utf16CodePointAt` function to see how surrogates are handled:

https://github.com/HaxeFoundation/haxe/blob/096b447deeb0cf240b1cf18548b8af1cbba9c061/std/StringTools.hx#L603-L609